griptape/drivers/prompt/anthropic_prompt_driver.py

```diff
@@ -21,9 +18,9 @@ class AnthropicPromptDriver(BasePromptDriver):
         tokenizer: Custom `AnthropicTokenizer`.
     """
 
-    api_key: str = field(kw_only=True, metadata={"serializable": True})
+    api_key: Optional[str] = field(kw_only=True, default=None, metadata={"serializable": True})
     model: str = field(kw_only=True, metadata={"serializable": True})
-    client: Anthropic = field(
+    client: Any = field(
         default=Factory(
             lambda self: import_optional_dependency("anthropic").Anthropic(api_key=self.api_key), takes_self=True
```

Comment:
is the `import_optional_dependency` still required here? it seems we removed the optional `anthropic` dependency import. Should this be something else?

Reply:
Yes, the only this we removed is the type hint which causes issues with serialization.